storybook/src/Omar/Atoms/Inputs/Inputs.js

```diff
@@ -0,0 +1,75 @@
+import { React, useState } from "react";
```

Comment:
- The home svg is defaulted, it should be passed as a prop to change it dynamically.
- No right icon applied, please add it. and make it dynamic.
- Please remove the text when clicking on x icon

Reply:
All done.

Reply:
Added the default case without any left/right icons. and the spacing as well.